datafusion/physical-expr/src/functions.rs

```diff
@@ -83,26 +83,6 @@ pub fn create_physical_expr(
     )))
 }
 
-#[cfg(feature = "encoding_expressions")]
-macro_rules! invoke_if_encoding_expressions_feature_flag {
-    ($FUNC:ident, $NAME:expr) => {{
-        use crate::encoding_expressions;
-        encoding_expressions::$FUNC
-    }};
-}
-
-#[cfg(not(feature = "encoding_expressions"))]
-macro_rules! invoke_if_encoding_expressions_feature_flag {
-    ($FUNC:ident, $NAME:expr) => {
-        |_: &[ColumnarValue]| -> Result<ColumnarValue> {
-            internal_err!(
-                "function {} requires compilation with feature flag: encoding_expressions.",
-                $NAME
-            )
-        }
-    };
-}
-
 #[cfg(feature = "crypto_expressions")]
```

Comment:
The other existing feature flags such as `crypto_expressions` are obvious candidates to extract into datafusion-functions